### ISSUE TITLE
[Stacks/prod] RuntimeError: There was a problem fetching http://imageserver-prod.stanford.edu/iiif/2/sv%2F051%2Fyq%2F8509%2FBranner_00017231_0001.jp2/info.json. Server returned 403

### ISSUE THREAD

**Issue by cbeer** [sul-dlss/stacks#240]
## Backtrace

    line 46 of [PROJECT_ROOT]/app/services/iiif_metadata_service.rb: retrieve
    line 52 of [PROJECT_ROOT]/app/services/iiif_metadata_service.rb: json
    line 33 of [PROJECT_ROOT]/app/services/iiif_metadata_service.rb: image_width

[View full backtrace and more info at honeybadger.io](https://app.honeybadger.io/projects/49299/faults/34740013)

**Comment by cbeer** [comment]
We're seeing this error for files that are shelved with incorrect file permissions (usually `640`). This means our image server cannot access them. I've sent to @blalbrit a list of 111k files with this problem.

**Comment by blalbrit** [comment]
argo/#977 - starting from objects impacted by this.

**Comment by jcoyne** [comment]
@blalbrit one of the objects in this group is https://purl.stanford.edu/sv051yq8509.xml.  It's not dark is it?

Rights looks like this:
```xml
  <rightsMetadata>
    <access type="discover">
      <machine>
        <world/>
      </machine>
    </access>
    <access type="read">
      <machine>
        <world/>
      </machine>
    </access>
    <use>
      <human type="useAndReproduction">Theses courtesy of Stanford University Libraries.  If you have questions, please contact the Branner Earth Science Library &amp;amp; Map Collections at brannerlibrary@stanford.edu.</human>
    </use>
  </rightsMetadata>
```

I'm not sure how sul-dlss/argo#977 applies.

**Comment by mejackreed** [comment]
Is perhaps `xp490xd1371` also part of this? I just went to the DRMC and found many items that were returning 403's incorrectly.

**Comment by jcoyne** [comment]
Looks like Honeybadger is lumping the 500 responses with the 400 responses.  I'm seeing this in the imageserver-c log for one occurance:

```
09:31:59.651 [ajp-apr-8009-exec-4] ^[[1;31mERROR^[[0;39m ^[[36mo.r.C.WebApplication^[[0;39m - Exception or error caught in server resource
org.restlet.resource.ResourceException: Internal Server Error (500) - The server encountered an unexpected condition which prevented it from fulfilling the request
...
Caused by: edu.illinois.library.cantaloupe.cache.CacheException: Unable to create directory: /image-cache/cantaloupe/info/90/5a/40
        at edu.illinois.library.cantaloupe.cache.FilesystemCache.put(FilesystemCache.java:964)
        at edu.illinois.library.cantaloupe.resource.AbstractResource.getOrReadInfo(AbstractResource.java:611)
        at edu.illinois.library.cantaloupe.resource.iiif.v2.InformationResource.doGet(InformationResource.java:108)
        at sun.reflect.GeneratedMethodAccessor23.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.restlet.resource.ServerResource.doHandle(ServerResource.java:523)
        ... 65 common frames omitted
Caused by: java.io.IOException: Unable to create directory: /image-cache/cantaloupe/info/90/5a/40
        at edu.illinois.library.cantaloupe.cache.FilesystemCache.put(FilesystemCache.java:949)
        ... 71 common frames omitted
```

**Comment by blalbrit** [comment]
@jcoyne - correct sv051yq8509 should not be throwing this problem. sul-dlss/argo#977 applies because I was given a list of 100K objects to "fix", but a percentage of them appear to be artifacts from the problem that argo#977 fixed (which went unfixed for nearly 5 years... so objects that were set to dark remained on stacks). I'm just trying to get a handle on which objects are actually a problem at this point.

**Comment by blalbrit** [comment]
@mejackreed xp490xd1371 seems to display fine in Mirador, though in a degraded form? That's weird.

![screenshot 2017-09-22 10 23 43](https://user-images.githubusercontent.com/796990/30756659-755fe762-9f80-11e7-8dfa-4c12acb65ba8.png)

**Comment by blalbrit** [comment]
After removing dupes (multiple files per object), I'm left with just under 20K objects that are either dark or have permission errors. Going to prune the dark objects out of stacks first and then fix the legit file permission errors.

**Comment by blalbrit** [comment]
5473 objects are dark and still on stacks - pruning those now.

**Comment by aaron-collier** [comment]
Addressed by #284.
